cmd/svcat/class/get_cmd.go

```diff
@@ -0,0 +1,98 @@
+/*
+Copyright 2018 The Kubernetes Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package class
+
+import (
+	"github.com/kubernetes-incubator/service-catalog/cmd/svcat/command"
+	"github.com/kubernetes-incubator/service-catalog/cmd/svcat/output"
+	"github.com/kubernetes-incubator/service-catalog/pkg/apis/servicecatalog/v1beta1"
+	"github.com/spf13/cobra"
+)
+
+type getCmd struct {
```

Comment:
It was a deliberate choice to keep as closely as possible to kubectl's grammar, which drove the choice not only of the verbs get and describe, but also using the same "binary verb noun" pattern.

It makes for a much easier learning curve when people's default instincts honed from using kubectl _just work_ with svcat. The output of svcat's get and describe were patterned after kubectl's output (though I didn't take it all the way due to time constraints).

I want to line-up svcat so that eventually when get and describe work well natively in kubectl, we can delegate those commands back to kubectl. Ideally resulting in as as few changes as possible for the end user, so they can seamlessly switch from `svcat get broker foo` to `kubectl get broker foo`.